<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Оптимизация траектории движения роя БЛС</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="showResultsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>961</width>
      <height>651</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="obstaclesMapOpen"/>
    <addaction name="obstaclesMapGenerate"/>
    <addaction name="separator"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Оптимизация</string>
    </property>
    <addaction name="setParams"/>
    <addaction name="startOptimization"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Результаты</string>
    </property>
    <addaction name="showResults"/>
    <addaction name="saveResults"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="aboutProgram"/>
    <addaction name="aboutQt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="obstaclesMapOpen"/>
   <addaction name="obstaclesMapGenerate"/>
   <addaction name="setParams"/>
   <addaction name="startOptimization"/>
   <addaction name="showResults"/>
   <addaction name="saveResults"/>
  </widget>
  <action name="obstaclesMapOpen">
   <property name="text">
    <string>Открыть карту препятствий</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="obstaclesMapGenerate">
   <property name="text">
    <string>Сгенерировать случайную карту препятствий</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="setParams">
   <property name="text">
    <string>Установить параметры оптимизации</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="startOptimization">
   <property name="text">
    <string>Начать оптимизацию</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="showResults">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Показать графические результаты оптимизации</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="saveResults">
   <property name="text">
    <string>Сохранить графические результаты оптимизации</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="aboutProgram">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="aboutQt">
   <property name="text">
    <string>О библиотеке &amp;Qt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
